SmartDeviceLink/private/SDLVoiceCommandManager.m

```diff
@@ -151,7 +151,7 @@ - (void)sdl_updateIdsOnVoiceCommands:(NSArray<SDLVoiceCommand *> *)voiceCommands
 - (void)sdl_commandNotification:(SDLRPCNotificationNotification *)notification {
     SDLOnCommand *onCommand = (SDLOnCommand *)notification.notification;
 
-    for (SDLVoiceCommand *voiceCommand in self.voiceCommands) {
+    for (SDLVoiceCommand *voiceCommand in self.currentVoiceCommands) {
```

Comment:
Did you test this? It didn't fix the problem and the array you changed it to is empty at this point.

All I'm doing is setting two voice commands:

```objc
SDLVoiceCommand *vc1 = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@"Test 1"] handler:^{
    NSLog(@"VC 1 selected");
}];
SDLVoiceCommand *vc2 = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@"Test 2"] handler:^{
    NSLog(@"VC 2 selected");
}];
self.sdlManager.screenManager.voiceCommands = @[vc1, vc2];
```